src/Frame.js

```diff
@@ -1 +1,44 @@
-// Frame class to go here
\ No newline at end of file
+class Frame {
+  constructor () {
+    this._rolls = [];
+    this._bonus_score = 0;
+  }
+
+  isStrike() {
+    return (this._rolls[0] === 10);
+  }
+
+  isSpare() {
+    return (this._rolls[0] + this._rolls[1] === 10);
+  }
+
+  calcFrameTotal() {
+    return this._rolls.reduce((a, b) => a + b, 0) + this._bonus_score;
+  }
+
+  isFrameComplete() {
+    if (this.isStrike() && this._rolls.length === 1) { return true; }
+    else if (this._rolls.length === 2 ) {return true;}
+  }
```

Comment:
You function returns `true` or `undefined`. It would be cleaner to return `true` or `false`

Reply:
A test that looks at unhappy paths would find this.